models/models.js

```diff
@@ -20,4 +20,13 @@ exports.selectArticles = () => {
       .then(({ rows: articles }) => {
         return articles;
       });
-};
\ No newline at end of file
+};
+
+exports.selectArticleById = (articleId) => {
+  return db.query(`SELECT * FROM articles 
+  WHERE articles.article_id = $1;`, [articleId])
+    .then(({ rows: article }) => {
+      return article[0];
```

Comment:
After you build the test that checks for a id that is a number but not a valid one, please implement the relevant solution in the model 🙂